resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                <tr>
+                    <th>Scenario</th>
+                    <th>Request Payload</th>
+                    <th>Response</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><p>Request with a valid payload, LISA Manager reference number and account ID</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "1234567890"
+        }
+        </pre>
+                    </td>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">200 (OK)</code></p>
+        <pre class="code--block">
+        {
+          "status": 200,
+          "success": true,
+          "data": {
+            "message": "This account has been reinstated",
+            "accountId": "1234567890"
+          }
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request with a valid payload and account ID, but an invalid LISA Manager reference number</p><p class="code--block">lisaManagerReferenceNumber: A12345</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "1234567890"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter lisaManagerReferenceNumber in the correct format, like Z1234"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Request with a invalid payload</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "1234=5678"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+          {
+            "code": "BAD_REQUEST",
+            "message": "Bad Request",
+            "errors": [
+              {
+                "code": "INVALID_FORMAT",
+                "message": "Invalid format has been used",
+                "path": "/accountId"
+              }
+            ]
+          }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for an account that is open or active</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks